hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorClientTest.java

```diff
@@ -83,37 +82,7 @@ protected ClientConfig clientConfig() {
 
     @Test
     public void testTimeout_execute() {
-        checkTimeout(true);
-    }
-
-    @Test
-    public void testTimeout_fetch() {
```

Comment:
Maybe we should leave this one? I believe it tests different operation than above.